Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/ElementWrappers/WebElementWrapper.cs

```diff
@@ -89,33 +117,34 @@ public bool Stale
 
         public IEnumerable<IWebElementWrapper> FindSubElements(By locator, string caption)
         {
-            var elements = Assert.ShouldGet(() => Element.FindElements(locator));
-            return ElementsToWrappers(elements, caption);
+            try
+            {
+                return ElementsToWrappers(Assert.ShouldGet(() => Element.FindElements(locator)), caption);
+            }
+            catch (Exception e) when (e is NullReferenceException or InvalidOperationException)
+            {
+                NUnit.Framework.Assert.Fail($"Couldn't find elements with {caption}");
+                return null;
+            }
         }
 
         private IEnumerable<IWebElementWrapper> ElementsToWrappers(IEnumerable<IWebElement> elements, string caption)
         {
-            foreach (var element in elements)
-            {
-                var wrapper = new WebElementWrapper(() => element, caption, _driverService);
-                yield return wrapper;
-            }
+            return elements.Select(element => new WebElementWrapper(() => element, caption, Driver));
         }
 
-        protected IDriverService Driver => _driverService;
+        protected IDriverService Driver { get; }
 
         private bool AriaEnabled
         {
             get
             {
-                switch (Element.GetAttribute("aria-disabled"))
+                return Element.GetAttribute("aria-disabled") switch
```

Comment:
I am not sure `aria-disabled` is common stuff.

Reply:
Removed